### PR TITLE
{vis} [intel-2018a] PyQtGraph v0.10.0 - Python/3.6.4

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.10.0-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+source_urls = ['http://www.pyqtgraph.org/downloads/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c74597dd87c31987099bd0949a5207e5d58297bd8a663279562ca1cb334074fc']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('PyQt5', '5.9.2', versionsuffix),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.10.0-intel-2018a-Python-3.6.4.eb]: Please add `use_pip = True`

Comment (SimonPinches) [reply easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.10.0-intel-2018a-Python-3.6.4.eb]: @vanzod, I'll do this in a new commit now...

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/024c3d2d9ddea9e9ec94eaf1fd7ddde6 for a full test report.

**Review by vanzod — APPROVED** [review]

**Comment by vanzod** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**